### ISSUE TITLE
core dump at /core in official image

### ISSUE THREAD

**Issue by RRAlex** [hashicorp/docker-consul#36]
Hi,

there seems to be an 8MB core file left on the root of the consul container that... probably shouldn't be there! (you might want to check no credentials leaked?).

How to reproduce:
docker run --rm -it consul ls -l /core

**Comment by slackpad** [comment]
Hi @RRAlex I'm not able to reproduce this:

```
$ docker run --rm -it consul ls -l /core
Unable to find image 'consul:latest' locally
latest: Pulling from library/consul
6c123565ed5e: Pull complete
a47353316dce: Pull complete
8f0cd2ab4ac5: Pull complete
0ba1d99c6d68: Pull complete
78f07e451a73: Pull complete
Digest: sha256:be5c652e6fdc77b57fba60517c1a714cca1c98edb119ab6bebed0ace120127ac
Status: Downloaded newer image for consul:latest
ls: /core: No such file or directory
```

Is it possible you've made a local image named "consul" or something like that?

**Comment by RRAlex** [comment]
Sorry, I apparently checked too fast, even though the image was 11 weeks old, I must have tried something...
TL;DR: downloaded again and it's not there. My fault, tired... sorry again! :)
